news/instructions.md

```diff
@@ -139,6 +159,17 @@ but the label can be interesting unless it's already part of the description.
 
 Use regular markdown for the image so it can be appropriately rendered and styled.
 
+### Contributor mentioning
+
+The individuals that have contributed to a noteworthy change _can_ be mentioned as contributors of it.
+Contributors should be mentioned below the headline as suggested in the [template](#template).
+If anybody else than the author of the N&N entry is listed, that person's consent should be obtained.
+Every individual who made code contributions to the described change should be mentioned as contributors (if consented).
+If too many people contributed to a change and the list becomes too long it can be [collapsed](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-collapsed-sections)
+or no contributor could be mentioned at all.
+The name of each contributor can be backed by a link to that person's profile and be accompanied by a little profile picture as suggested in the [template](#template).
```

Comment:
This sentence is currently very vague, but I think we should restrict the link here and not allow arbitrary sites.
The GH profile or a linked-in profile would seems suitable to me, but already a link to a private website could be difficult as the page might change it's content in the future for various reasons and could contain content that's not serious and we don't want to link to.

So we could either introduce an allow-list or specify relatively precisely what's allowed and what's not.